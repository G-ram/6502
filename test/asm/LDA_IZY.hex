a9 40 8d 04 00 a9 02 8d 06 00 a0 02 a9 00 b1 06
09 01 8d 00 01 
