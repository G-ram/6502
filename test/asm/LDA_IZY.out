A: 41
X: 00
Y: 02
P: 00010000
S: 20
PC: 615
100 : 41
101 : 00
